manifests/helper.pp

```diff
@@ -0,0 +1,41 @@
+# manage a helper
```

Comment:
please add puppet-strings style documentation for the parameters.

Reply:
Done. I rebased on top of master and changed commit message to credit @jay7x , let me know if there is a better way